cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,12 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://react-redux.realworld.io');
+    cy.get(':nth-child(2) > .nav-link').click();
+    cy.get(':nth-child(1) > .form-control').type('tester111@test.net');
+    cy.get(':nth-child(2) > .form-control').type('tester111');
+    cy.get('.btn').click();
+    cy.get(':nth-child(4) > .nav-link')
```

Comment:
The check for the username is not correct. The username used for login is 'tester111' but the assertion is checking for 'tester1111'. Make sure the assertion is checking for the correct username.